examples/observability_at_event_routing/variables.tf

```diff
@@ -89,7 +89,7 @@ variable "metadata_region_backup" {
 variable "permitted_target_regions" {
   type        = list(string)
   description = "List of regions where target can be defined."
-  default     = ["us-south", "eu-de", "us-east", "eu-gb", "au-syd"]
+  default     = ["us-south", "eu-de", "us-east", "eu-es", "eu-gb", "au-syd"]
```

Comment:
Not sure why this validation is here and not in the module. Can we move it to https://github.com/terraform-ibm-modules/terraform-ibm-observability-instances/blob/067395ffcf7087621af64bdd9414d4d76cd2b9ec/modules/activity_tracker/variables.tf#L180-L190 ?